f2/src/components/EditButton/index.js

```diff
@@ -7,9 +7,9 @@ import { useStateValue } from '../../utils/state'
 
 export const EditButton = ({ path, label }) => {
   const { i18n } = useLingui()
-  const [{ formData }] = useStateValue()
+  const [{ submitted }] = useStateValue()
 
-  return formData.submitted ? null : (
+  return submitted ? null : (
```

Comment:
Just a quick question, what is the difference between formData and submitted again?

Reply:
`formData` holds the forms the user has filled out. `submitted` is an extra boolean variable that the app uses to know whether the user has already submitted or not (so, for example, if they navigate back from the thank you page the app doesn't let them submit the same report again)